features/admin.ask-password-flow-builder.v1/components/resource-property-panel/steps/execution/ask-password-configurations.tsx

```diff
@@ -0,0 +1,656 @@
+     */
+    isConnectorEnabled?: boolean;
+    /**
+     * Specifies if the form is submitting
+     */
+    isSubmitting?: boolean;
+}
+
+/**
+ * Ask Password Configuration Form.
+ *
+ * @param props - Props injected to the component.
+ * @returns Functional component.
+ */
+export const AskPasswordConfigurations: FunctionComponent<AskPasswordConfigurationsPropsInterface> = (
+    props: AskPasswordConfigurationsPropsInterface
+): ReactElement => {
+
+    const {
+        connector,
+        readOnly,
+        ["data-componentid"]: componentId = "ask-password-edit-form"
+    } = props;
+
+    const { t } = useTranslation();
+
+    const {
+        setIsInvitedUserRegistrationConfigUpdated,
+        setInvitedUserRegistrationConfig
+    } = useAskPasswordFlowBuilder();
+
+    const [ initialConnectorValues, setInitialConnectorValues ]
+        = useState<AskPasswordFormValuesInterface>(undefined);
+    const [ isInviteUserToSetPasswordEnabled, setIsInviteUserToSetPasswordEnabled ]= useState<boolean>(false);
+    const [ isUpperCaseEnabled, setIsUpperCaseEnabled ] = useState<boolean>(false);
+    const [ isLowerCaseEnabled, setIsLowerCaseEnabled ] = useState<boolean>(false);
+    const [ isNumericEnabled, setIsNumericEnabled ] = useState<boolean>(false);
+    const [ askPasswordOption, setAskPasswordOption ] = useState<string>(VerificationOption.EMAIL_LINK);
+    const [ expiryTime, setExpiryTime ] = useState<string>("");
+    const [ otpLength, setOtpLength ] = useState<string>("");
+    const [ enableAccountActivationEmail, setEnableAccountActivationEmail ] = useState<boolean>(false);
+    const [ enableAccountLockOnCreation, setEnableAccountLockOnCreation ] = useState<boolean>(false);
+    const [ updatedConfigs, setUpdatedConfigs ] = useState<AskPasswordFormUpdatableConfigsInterface>(undefined);
+    const [ otpAccordionExpanded, setOtpAccordionExpanded ] = useState<boolean>(false);
+
+    const showSmsOtpAskPasswordFeatureStatusChip: boolean =
+            useSelector((state: AppState) => state?.config?.ui?.showSmsOtpAskPasswordFeatureStatusChip);
+
+    /* Radio options for ask password */
+    const EMAIL_ASK_PASSWORD_RADIO_OPTIONS: RadioChild[] = [
+        {
+            label: "extensions:manage.serverConfigurations.userOnboarding." +
+                    "inviteUserToSetPassword.form.fields.emailAskPasswordOptions.emailLink.label",
+            value: VerificationOption.EMAIL_LINK
+        },
+        {
+            label: "extensions:manage.serverConfigurations.userOnboarding." +
+                    "inviteUserToSetPassword.form.fields.emailAskPasswordOptions.emailOtp.label",
+            value: VerificationOption.EMAIL_OTP
+        },
+        {
+            label: "extensions:manage.serverConfigurations.userOnboarding." +
+                    "inviteUserToSetPassword.form.fields.emailAskPasswordOptions.smsOtp.label",
+            value: VerificationOption.SMS_OTP
+        }
+    ];
+
+    // Sync expansion with askPasswordOption
```

Comment:
```suggestion
    // Sync expansion with askPasswordOption.
```
Check other places as well

Reply:
Addressed the comment in commit [9fd9c8a](https://github.com/wso2/identity-apps/pull/9215/commits/9fd9c8a380f103b42e07792cd94865e0caaa8ca5)